hr_recruitment_candidate_multi_applicant/models/hr_applicant.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Author: Denis Leemann
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class HrApplicant(models.Model):
+    _inherit = 'hr.applicant'
+
+    same_candidate_application_ids = fields.Many2many(
+        'hr.applicant',
+        string='Same candidates',
+        compute='_compute_same_candidate_application_count',
+        readonly=True,
+    )
+    same_candidate_application_count = fields.Integer(
+        'Number of similar candidates',
+        compute='_compute_same_candidate_application_count',
+        readonly=True,
+    )
+
+    @api.depends('email_from', 'partner_phone')
+    def _compute_same_candidate_application_count(self):
+        for applicant in self:
+            domain = ['|', ('active', '=', False), ('active', '=', True),
+                      '|', '&', ('email_from', '=', applicant.email_from),
+                      ('email_from', '!=', False), '&',
+                      ('partner_phone', '=', applicant.partner_phone),
+                      ('partner_phone', '!=', False)]
+            if not isinstance(applicant.id, models.NewId):
+                domain = [('id', '!=', applicant.id)] + domain
+            same_app = self.search(domain)
```

Comment:
use  `active_test = False`in the search and remove the active = True or active = False in the domain.